subprojects/distributions-dependencies/distributions-dependencies.gradle.kts

```diff
@@ -118,7 +118,7 @@ dependencies {
         api(libs.maven3WagonHttpShared) { version { strictly(mavenWagonVersion) }}
         api(libs.maven3WagonProviderApi) { version { strictly(mavenWagonVersion) }}
         api(libs.minlog)                { version { strictly("1.2") }}
-        api(libs.nativePlatform)        { version { strictly("0.22-milestone-4") }}
+        api(libs.nativePlatform)        { version { strictly("0.22-snapshot-20200715094808+0000") }}
```

Comment:
Can you publish a snapshot from native platform's `master` branch?

Reply:
No, from my branch. Is this important?

Reply:
I find it easier if the snapshots we use are from native platform's `master` branch, so it is easier to check what the diff between two snapshots is. I think the rule should be that we use snapshots from `master` on Gradle's `master` branch.